xbmc/pvr/addons/PVRClients.h

```diff
@@ -80,23 +80,25 @@ namespace PVR
      * @brief Update add-ons from the AddonManager
      * @param changedAddonId The id of the changed addon, empty string denotes 'any addon'.
```

Comment:
Please add documentation for new parameter `changedInstance`

@param changedInterface The Identifier of the changed add-on instance